### DISCUSSION TITLE
Question about usage of `CrossValidation`

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#4080] [Q&A] [answered]
Hello MONAI team,

Recently I'm trying to use [`CrossValidation` class](https://github.com/Project-MONAI/MONAI/blob/b5e7fe1dfe0900705345671f4d532b6c18fea29a/monai/apps/datasets.py#L382). But after browsing the doc and the source code I get confused about its usage. I don't understand the reason it requires a `_split_datalist` API for `dataset_cls`, since I can't find where it's invoked and it is also overridden by the derived class `_NsplitsDataset` nested in the implementation of `get_dataset`. Could you please suggest some hints on this or add a more detailed example in the doc?

Thanks in advance!

**Comment by Nic-Ma** [comment, marked as answer]
Hi @function2-llx ,

Thanks for your interesting with this module, I think you can refer to this tutorial for more usage details:
https://github.com/Project-MONAI/tutorials/blob/master/modules/cross_validation_models_ensemble.ipynb

Thanks.

**Comment by function2-llx** [comment]
Hello @Nic-Ma ,

Thanks for the reply! Now I get the intended usage of this class, though I just feel like there should be a more natural and clear way to do this. Maybe I will be back later when I have some more time to discuss and help to improve.
